docs/contributing.md

```diff
@@ -126,6 +126,8 @@ If you're prepared to contribute by submitting a pull request, ensure you have m
 - `cargo check` has been run.
 - `cargo sqlx prepare` has been run.
 
+> Note: If you encounter issues with `sqlx` saying 'no queries found' after running `cargo sqlx prepare`, you may need to ensure the installed version of `sqlx-cli` matches the current version of `sqlx` used [in labrinth](https://github.com/modrinth/labrinth/blob/master/Cargo.toml).
```

Comment:
I'm not sure if this is really the proper solution, but it's what got `cargo sqlx prepare` to work for me (based on https://github.com/launchbadge/sqlx/issues/788#issuecomment-1311937003)